src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Calendar, luxonLocalizer } from 'react-big-calendar';
+import { CourseEvent, Schedule, scheduleCalendarResources } from 'services/schedule';
+import './ScheduleCalendar.css';
+import { DateTime } from 'luxon';
+
+const localizer = luxonLocalizer(DateTime, { firstDayOfWeek: 1 });
+
+type Props = {
+  schedule: Schedule;
+  onSelectEvent: (event: CourseEvent) => void;
+};
+
+const GordonScheduleCalendar = ({ schedule, onSelectEvent }: Props) => {
+  const dayStart = new Date();
+  dayStart.setHours(8, 0, 0, 0);
```

Comment:
Are there no classes that start before 8am, labs or otherwise? I assume this is limiting the hours displayed on the calendar.

Reply:
I did not implement this, so I don't know without investigating. I know that this was considered when they updated the schedule this summer.